src/ringct/rctTypes.h

```diff
@@ -490,5 +545,6 @@ VARIANT_TAG(json_archive, rct::mgSig, "rct_mgSig");
 VARIANT_TAG(json_archive, rct::rangeSig, "rct_rangeSig");
 VARIANT_TAG(json_archive, rct::boroSig, "rct_boroSig");
 VARIANT_TAG(json_archive, rct::rctSig, "rct_rctSig");
+VARIANT_TAG(json_archive, rct::Bulletproof, "rct_bulletproof");
```

Comment:
Perhaps `src/serialization/json_object` needs to be updated as well? Just a guess.

Reply:
Hmm. I guess, I don't know about this one... I think it's a new 0mq one. I'll fix it up too.